Create sensoterra.markdown (#33276)
home-assistant/home-assistant.github.io: source/_integrations/sensoterra.markdown

---
title: Sensoterra
description: Instructions on how to integrate your Sensoterra probes with Home Assistant.
ha_release: 2024.7
ha_category:
  - Sensor
ha_iot_class: Cloud Poll
ha_config_flow: true
ha_domain: sensoterra
ha_platforms:
  - sensor
ha_codeowners:
  - '@markruys'
ha_integration_type: integration
ha_quality_scale: silver
---

> This integration communicates with your [Sensoterra](https://sensoterra.com) soil moisture probes using the public [Sensoterra Customer API](https://monitor.sensoterra.com/api/v3/) via HTTPS.

Reviewer: ```suggestion
This {% term integration %} communicates with your [Sensoterra](https://sensoterra.com) soil moisture probes using the public [Sensoterra Customer API](https://monitor.sensoterra.com/api/v3/) via HTTPS.
```

> ## Pre-requisites

Reviewer: ```suggestion
## Prerequisites
```

Soil moisture probes can be purchased at Sensoterra.com. Create a free account in the Sensoterra app and register your probes. The Sensoterra Home Assistant integration needs your credentials to obtain a token to access your probe data.

> 

Reviewer: ```suggestion

{% include integrations/config_flow.md %}

```

## Sensors

The integration provides the following sensors for each probe:

- Soil moisture (either in volumetric-% or in SI)

> - Temperature

Reviewer: What is the unit used here? degrees Celsius?

Reviewer: Drafting this PR as there is a review comment. Please undraft it again, once the PR is ready.

- LoRaWAN signal strength (RSSI level)
- Battery level (100, 50 or 10% approximations)
- Last seen timestamp

> Typically, soil moisture probes measure once an hour. The HA integration polls this data each 15 minutes.

Reviewer: ```suggestion
Typically, soil moisture probes measure once an hour. The Home Assistant integration polls this data every 15 minutes.
```

---
title: Sensoterra
description: Instructions on how to integrate your Sensoterra probes with Home Assistant.
ha_release: 2024.10
ha_category:
  - Sensor
ha_iot_class: Cloud Poll
ha_config_flow: true
ha_domain: sensoterra
ha_platforms:
  - sensor
ha_codeowners:
  - '@markruys'
ha_integration_type: integration

> ha_quality_scale: silver

Reviewer: ```suggestion
```
Removing this, as the quality scale is not defined in the manifest.
